components/modal/modal.md

```diff
@@ -130,8 +130,7 @@ Basic Modal:
             <Modal
               active={state.isOpenModalWithCard}
               isSmall
-              onClose={(e) => {
-                e.persist();
```

Comment:
Probably will be better to use like this? `e && e.persist();`

Reply:
But why do we need this?

Reply:
Checked. For now it works fine without persist(). Thanks ;)